driver/normalizer/normalizer.go

```diff
@@ -87,20 +87,32 @@ var Normalizers = []Mapping{
 		CasesObj("case",
 			// common
 			Obj{
-				"importKind": String("value"),
 				"source":     Var("path"),
 			},
 			Objs{
 				// namespace
 				{
+					"importKind": String("value"),
 					"specifiers": ArrWith(Var("names"), Obj{
+						//uast.KeyType: Var("spec_type"),
 						uast.KeyType: String("ImportNamespaceSpecifier"),
 						uast.KeyPos:  Var("local_pos"),
 						"local":      Var("local"),
 					}),
 				},
+				// specific type
+				{
+					"importKind": String("type"),
+					"specifiers": ArrWith(Var("names"), Obj{
+						uast.KeyType: String("uast:Alias"),
```

Comment:
Please use `UASTType(uast.Alias{}, Obj{` instead (cannot be used above since native types are not registered this way). It will make sure that all fields are checked correctly (including `@pos` that is missing in this annotation).

Reply:
It's the `KeyType`, only used to discriminate between cases in the `CaseObj` and its values are not used in the node converted to `uast:Import` but I still should add a full `Obj{}`?

Reply:
Changed.